tests/acceptance/00_basics/environment/proc-net-functions.cf.sub

```diff
@@ -0,0 +1,30 @@
+# you can run this test directly with
+
+# CFENGINE_TEST_OVERRIDE_PROCDIR=`pwd`/00_basics/environment/proc testall 00_basics/environment/proc-net-functions.cf.sub
+
+body common control
+{
+    inputs => { "../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+    version => "1.0";
+}
+
+bundle agent test
+{
+  vars:
+      "connections" data => network_connections();
+}
+
+bundle agent check
+{
+  vars:
+      "testname" string => regex_replace($(this.promise_filename), "\\.sub$", "", "");
+
+  methods:
+      "check"  usebundle => dcs_check_state(test,
+                                           "$(this.promise_filename).expected.json",
+                                           $(testname));
+      "check"  usebundle => dcs_check_state(test,
+                                           "$(this.promise_filename).expected.json",
+                                           $(this.promise_filename));
```

Comment:
At least protect it with a class you have to set, maybe `debug_proc` or something. The reason is that we've had a few cases of false negatives in the past, where the test passes, but it should have failed. I really want to minimize that possibility, and this would increase the risk.